packages/rum-core/src/domain/contexts/featureFlagContext.ts

```diff
@@ -69,7 +69,7 @@ export function startFeatureFlagContexts(
     addFeatureFlagEvaluation: (key: string, value: ContextValue) => {
       const currentContext = featureFlagContexts.find()
       if (currentContext) {
-        currentContext[key] = deepClone(value)
+        currentContext[key] = sanitize(value)
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**💬 suggestion:** ‏should we sanitize the key as well?

Reply:
👌 Done, but moved the code to rumPublicApi.ts